source/transfer_list.rst

```diff
@@ -657,5 +658,58 @@ such that the last ACPI table in this entry ends at offset
      - hdr_size
      - One or more ACPI tables.
 
+
+.. _tpm_evlog_entry:
+
+TPM info table entry layout (XFERLIST_TPM_INFO)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This entry type holds TPM info of the platform.
+
```

Comment:
I think adding a bit more information here would help provide context to readers.
Suggestion:

> This entry type holds TPM-related information for a platform.  The TPM event log info is a region containing a TPM event log as defined by TCG EFI Protocol Specification (https://trustedcomputinggroup.org/resource/tcg-efi-protocol-specification/).  The CRB info defines the address of a region of memory that has been carved out and reserved for use as a TPM Command Response Buffer interface.